Loop54.Shared/Loop54Settings.cs

```diff
@@ -9,13 +7,12 @@ namespace Loop54
     /// </summary>
     public class Loop54Settings
     {
-        /// <summary>
-        /// Constructor
-        /// </summary>
         /// <param name="endpoint">The endpoint of the Loop54 search engine. If you don't have this please contact customer support.</param>
-        public Loop54Settings(string endpoint)
+        /// <param name="apiKey">The API key authenticating you as a trusted caller. If you don't have this please contact customer support.</param>
+        public Loop54Settings(string endpoint, string apiKey = null)
```

Comment:
I think we should drop the default value of `apiKey` in the next major version to encourage the caller to pass that in, but for now I've added the default value to preserve source compatibility